sdk/dotnet/Pulumi.Automation/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,8 @@
+Pulumi.Automation.PluginInstallOptions
```

Comment:
Separate question, should API changes go straight into the `Shipped` file for a PR or is it correct to have them here in `Unshipped`?

Reply:
Sorry for the delay here, I'll check on that and get back to you. 

We have an internal release process doc https://github.com/pulumi/home/wiki/Producing-the-Pulumi-CLI---SDKs and it does not have any step for moving Unshipped->Shipped (an oversight perhaps). So I think it should go straight into the Shipped file for now.

CC @emiliza --^ alternative here is that we fixup our release process to do this moving of Unshipped to Shipped. Context: these text files are integrated with CI-based API surface checker in .NET so any changes to public API surface need to be marked in these files to be considered intentional.

Reply:
Sounds good the changes have been moved to `Shipped`